cordova-lib/spec-cordova/fixtures/plugins/sample-cocoapod-plugin-overlapping-dependency/RELEASENOTES.md

```diff
@@ -0,0 +1,141 @@
+<!--
```

Comment:
Lots more files to delete. All the files that aren't necessary for the test can be deleted. Releasenotes, readmes, licenses, etc. 

In fact, you can delete out all the native files that don't pertain to ios as well. Even delete those sections out of plugin.xml